stablehlo/dialect/StablehloOps.td

```diff
@@ -1460,7 +1460,8 @@ def StableHLO_BatchNormGradOp : StableHLO_Op<"batch_norm_grad", [NoSideEffect,
         "grad_offset"]>,
     AllShapesMatch<["operand", "grad_output"]>,
     AllElementTypesMatch<["operand", "grad_scale", "grad_offset"]>,
-    AllTypesMatch<["operand", "grad_operand"]>]> {
+    AllTypesMatch<["operand", "grad_operand"]>,
+    InferTensorType]> {
```

Comment:
If you expand the definition of this op, then it'll say `let results` twice. What does this mean?

Reply:
Removed the first line as sync with Sandeep offline. It was a typo.